### DISCUSSION TITLE
DKIM setup on namecheap

### DISCUSSION THREAD

**Discussion by datendieman** [simple-login/app#485] [Q&A]
Hello,

I've added a custom domain to my SL account and have successfully added the MX record, SPF, and DMARC. However, no matter what I do, I cannot get the DKIM to verify.

Just to confirm:

Host: dkim._domainkey
Value: dkim._domainkey.simplelogin.co.
TTL: automatic

Any ideas what I can do differently? (24 hours has already passed)

Thanks.

**Comment by nguyenkims** [comment]
Hi can you check the DKIM on a third party website, for example https://mxtoolbox.com/dkim.aspx, to check the DKIM record has been successfully updated everywhere?
